CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Delete the ability to explicitly specify multiple files, and have the effect
   of one file being specified be the same as the former `--dir-mode`. See
   [#16](https://github.com/uber/prototool/issues/16) for more details.
+- Remove protoc_include_wkt setting. This is always set to true.
```

Comment:
Stupid things:
- s/Remove/Delete/ to be consistent with above
- Put tickmarks around each option name ie `protoc_include_wkt`.

Reply:
Sure, sounds good!